pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/BatchIngestionConfig.java

```diff
@@ -74,4 +92,12 @@ public void setSegmentIngestionType(String segmentIngestionType) {
   public void setSegmentIngestionFrequency(String segmentIngestionFrequency) {
     _segmentIngestionFrequency = segmentIngestionFrequency;
   }
+
+  public void setConsistentDataPush(boolean consistentDataPush) {
```

Comment:
Should we trace this at info level? it would be nice to know this was set for sure via logs